support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -92,27 +98,44 @@ export function CheckoutNudgeContainer({
 
 	let title, subtitle, paragraph;
 
-	if (dynamic) {
+	const calcWeeklyAmount = (annualAmount: number, toNearest?: number) => {
```

Comment:
I know the codebase is a bit all over the place on this for historic reasons, but function definitions (`function calcWeeklyAmount(//etc)`) are preferable for new code.

Reply:
I've extracted this out of the component and made it a pure function that sits outside of it now, could be considered a utility/helper function that we could move elsewhere. Let me know what you think.